autotest/ogr/ogr_mssqlspatial.py

```diff
@@ -609,17 +609,21 @@ def test_binary_field_bcp():
     ds = ogr.GetDriverByName("GPKG").CreateDataSource(filename)
     src_lyr = ds.CreateLayer("test_binary_field", geom_type=ogr.wkbPoint)
     src_lyr.CreateField(ogr.FieldDefn("binfield", ogr.OFTBinary))
+    src_lyr.CreateField(ogr.FieldDefn("strfield", ogr.OFTString))
     f = ogr.Feature(src_lyr.GetLayerDefn())
     f.SetGeometry(ogr.CreateGeometryFromWkt("POINT(1 2)"))
     f.SetFieldBinaryFromHexString("binfield", "007FFF007FFF")
+    f["strfield"] = "some text"
     src_lyr.CreateFeature(f)
     f = ogr.Feature(src_lyr.GetLayerDefn())
     f.SetGeometry(ogr.CreateGeometryFromWkt("POINT(2 3)"))
-    f.SetFieldBinaryFromHexString("binfield", "7FFF007FFF00")
+    f.SetFieldBinaryFromHexString("binfield", "")
+    f["strfield"] = ""
```

Comment:
I would actually put content here in strfield, to make sure that a 0-size binfield doesn't mess up with non-empty dynamic field content after it

Reply:
done